src/services/auth.ts

```diff
@@ -0,0 +1,29 @@
+import { Auth } from 'aws-amplify';
+import { CognitoUser } from '@aws-amplify/auth';
+import { CognitoUserSession } from 'amazon-cognito-identity-js';
+
+export const signIn = async (
```

Comment:
If it doesn't need to be a variable, I rather prefer it declared as a proper function as it should be. I don't see you need arrow function properties as well.